src/components/Investigaciones/Investigaciones.jsx

```diff
@@ -1,62 +1,103 @@
 import styles from "./investigaciones.module.css";
 import Temas from '../Temas';
+import { Link } from "react-router-dom";
+import Icons from "../iconos/Icons";
 
-
-const tituloNotaPrincipal = "Titulo de la ultima invtegicacion con un largo máximo de esa cantidad de caracteres: esta cantidad.";
+const tituloNotaPrincipal = "La bonaerense en la mira: de una internación psiquiátrica a un doble femicidio";
     const bajadaNotaPrincipal = "Bajada de la ultima nota que se haya hecho donde el texto puede durar solamente esta cantidad máxima de caracteres";
     const fechaUltimoReporte = "xx/xx/xxxx";
-
+    const fotoNotaPrincipal = "../src/components/investigaciones/investigacion/Crisis_salud_Mental_Mapa3.jpg";
+    const fechaNotaPrincipal = "08/05/2024";
+    const autorxsNotaPrincipal = "Sofi y Pampa";
+    const ilusNotaPrincipal = "Panchopepe";
 
 
 const Investigaciones = () => {
 
     return(
         <>
 
+        <section className={styles.investContainer}>
+
+        <Link to="/investigacion/bonaerense-mira">
+
             <section className={styles.investNovedades}>
+             
              <section style= {Temas.invest2Tema} className={styles.notaPrincipal}>
-        
-                 <img src="" alt="" className={styles.fotoNotaPrincipal} />
+                 <img src={fotoNotaPrincipal} alt="" className={styles.fotoNotaPrincipal} />
                  <section className={styles.infoNotaPrincipal}>
-                 <h2 className={styles.tituloNotaPrincipal}>
+                    <h2 className={styles.tituloNotaPrincipal}>
                     {tituloNotaPrincipal}</h2>
-                    <h5>{bajadaNotaPrincipal}</h5>
+                    
+                    <section className={styles.infoBasica}>
+                     
+                       <section className={styles.autorxsContainer}>
+                        <div className={styles.autorxContainer}>
+                        <Icons className={styles.icon} icon="autorx"/>
+                    <h5>{autorxsNotaPrincipal}</h5>
+                        </div>
+                        <div className={styles.ilusContainer}>
+                        <Icons className={styles.icon} icon="ilus"/>
+                    <h5>{ilusNotaPrincipal}</h5>
+                        </div>
+                        </section>
+                  
+                    <h6 className={styles.fechaContainer}>{fechaNotaPrincipal}</h6>
+                  
                     </section>
+
+                    <h5 className={styles.bajadaContainer}>{bajadaNotaPrincipal}</h5>
+
+                 </section>
```

Comment:
this should be it's own component.
also you can already do a `articles.map()` here